frontend/app/src/components/history/events/tx/TransactionFormDialog.vue

```diff
@@ -0,0 +1,29 @@
+<script lang="ts" setup>
+withDefaults(
+  defineProps<{
+    loading?: boolean;
+  }>(),
+  {
+    loading: false
+  }
+);
+
+const { valid, openDialog, submitting, closeDialog, trySubmit } =
+  useHistoryTransactionsForm();
+
+const { t } = useI18n();
+</script>
+
+<template>
+  <big-dialog
+    :display="openDialog"
+    :title="t('transactions.dialog.add_tx')"
+    :primary-action="t('common.actions.save')"
+    :action-disabled="loading || submitting || !valid"
```

Comment:
Also here if we are validating when the button is pressed, we should remove the validity check.

Action should only be disabled when loading/submitting only